README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+<h1>My Homework #1</h1>
+
+My name is <strong>William McCullen</strong>.
+I am a 17 year old student and an incoming Senior in high school.
+Some of my favorite activities are:
+<ol>
+<li>Soccer</li>
+<li>Video Gaming</li>
+<li>Board/Card Games</li>
+</ol>
+Here is a link to a cool game that my friends and I play sometimes:
+[link](https://shellshock.io)
```

Comment:
I googled a YouTube video to find how to embed links in Markdown but I'm not sure why your code (seems to be written correctly) didn't display the link properly. You mentioned that you wanted to embed an image; I did this by following example code from https://guides.github.com/features/mastering-markdown/. Scroll down to "Examples" and click "Images". There is also more info further down the page. If it helps, you can take a look at my code (last line). I tried pasting the code here but it automatically see the image so you wouldn't be able to see the code.